frontend/app/component/form/pollingstation_save/PollingStationSaveForm.tsx

```diff
@@ -34,40 +54,80 @@ export function PollingStationSaveForm() {
   return (
     <Form onSubmit={handleSubmit} id="check_save_form">
       <h2>Controleren en opslaan</h2>
+      <section className="md">
+        {!summary.hasBlocks && summary.countsAddUp && (
+          <p className="md">
+            De aantallen die je hebt ingevoerd in de verschillende stappen spreken elkaar niet tegen. Er zijn geen
+            blokkerende fouten of waarschuwingen.
+          </p>
+        )}
+        {summary.hasBlocks && summary.countsAddUp && (
+          <>
+            <p className="md">
+              De aantallen die je hebt ingevoerd in de verschillende stappen spreken elkaar niet tegen. Er zijn
+              waarschuwingen die moeten worden gecontroleerd. Controleer de openstaande waarschuwingen
```

Comment:
"Controleer de openstaande waarschuwingen" should be on a new paragraph according to the design